src/rocqproverorg_frontend/pages/about.eml

```diff
@@ -106,7 +106,7 @@ Layout.render
           The new name, "the Rocq Prover", honors Inria Rocquencourt, the original site where the prover was developed.
           It also alludes to the mythological bird Roc (or Rokh), symbolizing strength and not so disconnected to a rooster.
           Furthermore, the name conveys a sense of solidity, and its unintended connection to music adds a pleasant resonance.
-          The new name was chosen by the <a href="<%s Url.governance_team "governance" %>#Core">Core team</a> after a poll of the users, see
+          The new name was chosen by the <a href="<%s Url.rocq_team (Some "governance") %>#Core">Core team</a> after a poll of the users, see
```

Comment:
You've removed the governance team, so this link is now broken (as well as a few others that I had spotted and fixed in #95).

Reply:
Ok. my bad :)

Reply:
Fixed